eclipse-collections-code-generator/src/main/resources/test/factory/primitiveObjectMapsTest.stg

```diff
@@ -0,0 +1,69 @@
+import "copyright.stg"
+import "primitiveLiteral.stg"
+
+isTest() ::= "true"
+
+targetPath() ::= "org/eclipse/collections/impl/map/primitive"
+
+fileName(primitive) ::= "<primitive.name>ObjectMapsTest"
+
+skipBoolean() ::= "true"
+
+class(primitive) ::= <<
+<body(primitive.type, primitive.name, primitive.wrapperName, primitive.charPrimitive)>
+>>
+
+body(type, name, wrapperName, charPrimitive) ::= <<
+<copyright()>
+
+package org.eclipse.collections.impl.map.primitive;
+
+import org.eclipse.collections.impl.factory.primitive.<name>ObjectMaps;
```

Comment:
We don't want to be importing the `impl` ObjectMaps here since we're generating new ObjectMaps factories under the `org.eclipse.collections.api.factory.primitive` package in this PR. Want to be testing those factories.

There are some examples of testing factories in the other template files under this package like: https://github.com/eclipse/eclipse-collections/blob/master/eclipse-collections-code-generator/src/main/resources/test/factory/primitiveStacksTest.stg

You can take a look there to see how the factories are tested as well as take a look at the various assertions used. There are a few factory testing template files under `src/main/resources/test/factory` to check out.

Reply:
Thanks for reviewing @rinatigati ... @Desislav-Petrov the file linked above shows the strategy we have used for testing both the api and impl versions of the factory classes for primitive Stacks. We've used the same strategy across other factory classes as well.

Reply:
Ah sry - think I finally got my head around it. So the idea here is that we get the factories obtained via the api module which will be calling the ServiceLoader.. which makes sense as this is how these will be obtained anyway. In fact, after re-working the test it caught a bug:) Thanks for the explanations